tutorials/understanding-authz/index.md

```diff
@@ -24,25 +24,25 @@ Before you start the tutorial, you need to install the `simd` binary.
 Clone the `cosmos-sdk` repository:
 
 ```sh
-$ git clone https://github.com/cosmos/cosmos-sdk
+git clone https://github.com/cosmos/cosmos-sdk
```

Comment:
I think we should keep these `$` throughout all the content when it is a shell script.

Reply:
I am using the `markdownlint` vscode extension, which follows the rules set in `.markdownlint.json` of this repository.

Removing the `$` sign is per the MD014 rule: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md014---dollar-signs-used-before-commands-without-showing-output

Reply:
I'd say don't make this change in this PR to keep a unity of style with all the other files.

Then open another PR where:

* you add the VSCode extension in `.vscode/extensions.json` (to add it, click suggest as per the screenshot)
* you add a short example effect of it on a file. 

Then we can discuss that and the content of `.markdownlint.json` with @coldice too.

Reply:
Sure!

Reply:
And here is the screenshot
![Screenshot 2022-07-13 at 17 15 46](https://user-images.githubusercontent.com/5309374/178742524-9c3aff05-cacc-42d0-8fb1-362a43429d9c.png)

Reply:
Created https://github.com/cosmos/sdk-tutorials/pull/1167 👍🏾